src/dashboard/components/navbar_component.py

```diff
@@ -99,3 +99,14 @@ def navbar_component() -> Component:
 def update_navbar(path_name: str) -> list[Component]:
     """Update the selected navbar item based on the current url."""
     return generate_navbar_items(dash.page_registry, path_name)
+
+
+@callback(
+    Output("main-navbar", "className"), Input("url", "pathname"), State("main-navbar", "className")
+)
+def show_navbar(path_name: str, class_name: str) -> list[Component]:
```

Comment:
MyPy fails because return type is `list[Component]` but callback output is className which is type `str`.

Reply:
Resolved :+1: